README.md

```diff
@@ -32,6 +32,7 @@ type FramebusOptions = {
   origin?: string, // default: "*"
   channel?: string, // no default
   verifyDomain?: (url: string) => boolean, // no default
+  limitBroadCastToOrigin?: boolean, // default false
```

Comment:
```suggestion
```

Reply:
Had those locally while working on the functional test, but since we're gonna keep it opt-in will resolve this.